dds/idl/be_util.cpp

```diff
@@ -64,6 +70,9 @@ be_util::prep_be_arg (char *arg)
   } else if (0 == ACE_OS::strncasecmp(arg, WB_PCH_INCLUDE, SZ_WB_PCH_INCLUDE)) {
     be_global->pch_include(arg + SZ_WB_PCH_INCLUDE);
 
+  } else if (0 == ACE_OS::strncasecmp(arg, WB_CPP_INCLUDE, SZ_WB_CPP_INCLUDE)) {
+    be_global->add_cpp_include(arg + SZ_WB_CPP_INCLUDE);
```

Comment:
I think we should add this to the usage message even if it's an "internal" option.  There is some precedent for that (see the text "special cases" in usage).

Reply:
OK, I'll add it.